### ISSUE TITLE
Only connect with usb

### ISSUE THREAD

**Issue by wizardxpita** [BimmerGestalt/AAIdrive#761]
OnePlus 11 after android 14 update I can connect only with usb
I try to remove phone and add again with bt but I can't see the app support on my id5

**Comment by j18hockey** [comment]
I think I had a similar issue. My S24+ (Android 14) would connect with Calls/Audio only. If I connected to the car via USB the apps option appeared and aaidrive populated apps. I managed to get it to connect with Apps over Bluetooth by doing the following (I'm not positive which was the magic bullet, but i did all of them):

- Developer Options -> Set Bluetooth AVRCP version to 1.3
- MyBMW app -> Allow Calendar, Contacts, Location, Nearby devices, Notifications, Photos and videos permissions
- MyBMWapp -> Turn off "Pause app activity if unused"

Id be curious if this works for you or if anyone has any input. I'm fairly certain the issue is with the MyBMW app connection to the car and once it is established aaidrive works great.
